gnmawsgr/views.py

```diff
@@ -64,9 +70,15 @@ def dispatch(self, request, *args, **kwargs):
 @has_group('AWS_GR_Restore')
 def re(request):
     from tasks import glacier_restore
+    from portal.vidispine.iitem import ItemHelper
 
     itemid = request.GET.get('id', '')
-    path = request.GET.get('path', '')
+
+    ith = ItemHelper()
+
+    res = ith.getItemMetadata(itemid)
+
+    path = metadataValueInGroup('ExternalArchiveRequest','gnm_external_archive_external_archive_path',res['item'])
```

Comment:
what is the difference between r() and re()? can we make them into the same method?